src/containers/my-cooperations/accept-cooperation-close/AcceptCooperationClosing.styles.ts

```diff
@@ -1,5 +1,25 @@
+import palette from '~/styles/app-theme/app.pallete'
+
 export const styles = {
   boldText: {
     fontWeight: 500
+  },
+  inputBox: {
+    display: 'flex',
+    flexDirection: 'column',
+    gap: '10px',
+    mb: '8px'
+  },
+  inputField: {
+    display: 'flex',
+    gap: '16px',
+    width: '100%',
```

Comment:
Probably if you remove flexbox from parrent container, you will not have to specify `width: 100%`, because of default block behaviour